### ISSUE TITLE
Carthage Framework not Built

### ISSUE THREAD

**Issue by ObjColumnist** [bryx-inc/BRYXBanner#8]
Hi,

When I add `BRYXBAnner`  to my cart file `github "bryx-inc/BRYXBanner"`it gets checked out but a framework isn't built, my guess is that the Scheme for the Framework isn't shared as I can't see it in the version that is checked out by Carthage?

**Comment by harlanhaskins** [comment]
Looking into this now -- thanks!

**Comment by alexpersian** [comment]
I'm having trouble replicating your issue locally @ObjColumnist, but this could very well be my mistake. Going through the project I double checked that the Scheme is shared. I also created a fresh test project with a Cartfile that only included this framework, and it built just fine. 

Here is where the Scheme is located. ![Scheme](http://i.imgur.com/LDAesu0.png)
this is within `BRYXBanner/Example/Pods/Pods.xcodeproj/xcshareddata/xcschemes`

Could you share some more information about your issue and the process you are using along with what you have in your Cartfile?

Cheers,

**Comment by ObjColumnist** [comment]
Sure,

So my Cart File looks like this:

```
github "Alamofire/Alamofire" >= 1.2.3
github "ObjColumnist/StaticTableView" "master"
github "ObjColumnist/StaticCollectionView" "master"
github "ObjColumnist/NetworkReachability" "master"
github "Present-Inc/mixpanel-iphone" >= 2.6.3
github "matthewpalmer/Locksmith" >= 1.2.2
github "bryx-inc/BRYXBanner"
```

But when I ran carthage update it cloned and checked out `BRYXBanner` but as you can see from the console never built the framework (hence my schema assumption):

```
Spencers-MacBook-Air:Demo smacdonald$ carthage update
*** Fetching Alamofire
*** Fetching StaticTableView
*** Fetching StaticCollectionView
*** Fetching NetworkReachability
*** Fetching mixpanel-iphone
*** Fetching Locksmith
*** Cloning BRYXBanner
*** Checking out Alamofire at "1.3.1"
*** Checking out mixpanel-iphone at "v2.6.3"
*** Checking out BRYXBanner at "0.2.2"
*** Checking out NetworkReachability at "2b74de3e9c2eb262ab8f1246a055ea977fe5e91f"
*** Checking out StaticCollectionView at "4f18c50982627c7565b64ed57096f6d0456264d8"
*** Checking out Locksmith at "1.2.2"
*** Checking out StaticTableView at "5f906990b5e891745411250a8692011cda5a05df"
*** xcodebuild output can be found in /var/folders/fq/q2tgw4mn7515828f8r7nrf8c0000gn/T/carthage-xcodebuild.ZUHTmf.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire OSX" in Alamofire.xcworkspace
*** Building scheme "Locksmith" in Locksmith.xcodeproj
*** Building scheme "NetworkReachability" in NetworkReachability.xcodeproj
*** Building scheme "StaticCollectionView" in StaticCollectionView.xcodeproj
*** Building scheme "StaticTableView" in StaticTableView.xcodeproj
*** Building scheme "Mixpanel" in Mixpanel.xcodeproj
```

**Comment by alexpersian** [comment]
Hmm, this is interesting. I'll look more into this. Thanks for the output information!

**Comment by ObjColumnist** [comment]
No problem,

As a side note, I imported Banner.swift directly into my project for now, but it imports Foundation rather than UIKit.

**Comment by alexpersian** [comment]
@ObjColumnist this is quite odd. I made a Cartfile that was copy/paste what you provided, then built it with `carthage update`, and this was what I received for console output.

```
Other-Barry:BannerTest alexpersian$ carthage update
*** Fetching BRYXBanner
*** Fetching Locksmith
*** Fetching mixpanel-iphone
*** Fetching NetworkReachability
*** Fetching StaticCollectionView
*** Fetching StaticTableView
*** Fetching Alamofire
*** Checking out Locksmith at "1.2.2"
*** Checking out StaticTableView at "5f906990b5e891745411250a8692011cda5a05df"
*** Checking out NetworkReachability at "2b74de3e9c2eb262ab8f1246a055ea977fe5e91f"
*** Checking out mixpanel-iphone at "v2.6.3"
*** Checking out StaticCollectionView at "4f18c50982627c7565b64ed57096f6d0456264d8"
*** Checking out Alamofire at "1.3.1"
*** Checking out BRYXBanner at "0.2.2"
*** xcodebuild output can be found in /var/folders/b8/srn7nr3563g9wb_jxfvh94xr0000gn/T/carthage-xcodebuild.xj4QZm.log
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire OSX" in Alamofire.xcworkspace
*** Building scheme "BRYXBanner" in Pods.xcodeproj
*** Building scheme "Locksmith" in Locksmith.xcodeproj
*** Building scheme "NetworkReachability" in NetworkReachability.xcodeproj
*** Building scheme "StaticCollectionView" in StaticCollectionView.xcodeproj
*** Building scheme "StaticTableView" in StaticTableView.xcodeproj
*** Building scheme "Mixpanel" in Mixpanel.xcodeproj
```

And all of the frameworks successfully built and are in `Carthage/build`

Have you tried running it again to see if it succeeds or throws any errors?

I'll do some more research tonight to see if I can make the Carthage support more robust.

**Comment by ObjColumnist** [comment]
No still the same even after deleting the Carthage Folder.

Il check that my version of carthage is up to date, to rule out that.

**Comment by ObjColumnist** [comment]
Upgrading Carthage Fixed it, I was on version 0.6.3 :)

The deployment target of the Framework is 8.3 though so would be great if that could be resolved.
